test/suites/edgex-config-provider/main_test.go

```diff
@@ -0,0 +1,156 @@
+package test
+
+import (
+	"edgex-snap-testing/test/utils"
+	"log"
+	"os"
+	"testing"
+	"time"
+)
+
+const (
+	platformSnap = "edgexfoundry"
+	provider     = "edgex-config-provider-example"
+)
+
+var services = []string{
+	"device-virtual",
+	/////////////////
```

Comment:
is this note?

Reply:
It was to separate device virtual from the remaining alphabetically sorted items. I removed and sorted all.